.gitignore

```diff
@@ -8,6 +8,7 @@ jspm_packages
 # production
 build
 storybook-static
+server/build
```

Comment:
Previously we added `node_modules` to gitignore and it was working for every node_modules. Is this `build` works different?

Reply:
No, that was my bad, sorry. After first creation of build I didn't get that low opacity effect like on other .gitignore files. But now everything seems to be ok.